system76/gaze18/default.nix

```diff
@@ -0,0 +1,37 @@
+{ config, lib, pkgs, ... }:
+{
+  imports = [
+    ../.
+  ];
+  boot.initrd.kernelModules = [ "nvidia" ];
```

Comment:
Part of the nvidia module.

```suggestion
```

Reply:
When I remove this, the nvidia drivers do not get loaded.

Reply:
It probably gets loaded, but later after the system is mounted. For intel drivers we also add them to the initrd by default. I don't know why we don't do this for nvidia as well by default.